### ISSUE TITLE
Some math.h functions are missed on WebAssembly backend

### ISSUE THREAD

**Issue by Kagami** [kripken/emscripten#6234]
```
$ llc --version
LLVM (http://llvm.org/):
  LLVM version 7.0.0svn
  Optimized build.
  Default target: x86_64-pc-linux-gnu
  Host CPU: sandybridge

  Registered Targets:
    wasm32 - WebAssembly 32-bit
    wasm64 - WebAssembly 64-bit
    x86    - 32-bit X86: Pentium-Pro and above
    x86-64 - 64-bit X86: EM64T and AMD64
```

```
$ cat 1.c
#include <math.h>
double test(double x) {
  double a = exp2(x);
  double b = log10(x);
  float c = log10f(x);
  double d = acos(x);
  return a + b + c + d;
}
```

```
$ emcc 1.c -s WASM=1 -s "EXPORTED_FUNCTIONS=['_test']" -o 1.js
warning: unresolved symbol: exp2
warning: unresolved symbol: log10
warning: unresolved symbol: log10f
$ grep 'missing function' 1.js
  Module['printErr']('missing function: exp2'); abort(-1);
  Module['printErr']('missing function: log10'); abort(-1);
  Module['printErr']('missing function: log10f'); abort(-1);
```

**Comment by kripken** [comment]
Thanks, a proposed fix is in #6235.

**Comment by Kagami** [comment]
Works now, thanks.
